centaur/src/main/resources/standardTestCases/globbing_behavior_with_requester_pays/globbing_behavior_with_requester_pays.wdl

```diff
@@ -0,0 +1,26 @@
+task globbing_behavior_task{
```

Comment:
ToL (because this comment is a little unfair) - would you consider making these all WDL 1.0s?

If you're using intelliJ you can just put `version 1.0` at the top and the highlighter will tell you what else to change (though looking at the tasks, I don't think it needs anything else changed)

Reply:
Added `version 1.0` to 2 WDLs 👍